vru_use_case_scripts/plot_detected_objects

```diff
@@ -169,6 +175,10 @@ sensor_map_position = parse_infrastructure_sensor_position_from_json(
 )
 
 figures_dir = Path("figures")
```

Comment:
Nit: we should pick one plots_dir or figures_dir. I am inclined to say plots dir since we use it in a lot of scripts